plugins/topology/src/__fixtures__/1-deployments.ts

```diff
@@ -794,6 +794,59 @@ export const mockKubernetesResponse = {
       },
     },
   ],
+  statefulSets: [
+    {
+      apiVersion: 'apps/v1',
+      kind: 'StatefulSet',
+      metadata: {
+        creationTimestamp: '2023-03-23T04:49:51Z',
+        generation: 1,
+        labels: {
+          'backstage.io/kubernetes-id': 'test-backstage',
+        },
+        name: 'example',
+        namespace: 'deb-test',
+        resourceVersion: '70480',
+        uid: 'c7afef06-4f6b-4582-9d3e-81ad69b10a75',
+      },
+      spec: {
+        podManagementPolicy: 'OrderedReady',
+        replicas: 3,
+        revisionHistoryLimit: 10,
+        selector: { matchLabels: {} },
+        serviceName: 'httpd',
+        template: {
+          metadata: {},
+          spec: {},
+        },
+        updateStrategy: {
+          type: 'RollingUpdate',
+          rollingUpdate: {},
+        },
+      },
+      status: {},
+    },
+  ],
+  cronJobs: [
+    {
+      apiVersion: 'batch/v1',
+      kind: 'CronJob',
+      metadata: {
+        creationTimestamp: '2023-03-23T04:50:35Z',
+        labels: {
+          'backstage.io/kubernetes-id': 'test-backstage',
+        },
+        name: 'example',
+        namespace: 'deb-test',
+        resourceVersion: '70698',
+        uid: 'a1ee9342-8bc7-4b6d-9271-9adc3766b9c7',
+      },
+      spec: {
+        concurrencyPolicy: 'Allow',
+        failedJobsHistoryLimit: 1,
+      },
+    },
+  ],
```

Comment:
shall we add mock data for Job as well?

Reply:
Done
![Screenshot 2023-03-23 at 7 30 35 PM](https://user-images.githubusercontent.com/22490998/227227743-42dfba0b-c379-40a9-8011-2018f84aa6c8.png)